.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+ contact_links:
```

Comment:
I think there is an extra space before `contact_links:`. It looks like every line below is shifted one extra space right, and I fear that'll invalidate the YAML file

Reply:
Good catch! Fixed.